hetmech/tests/hetnets.py

```diff
@@ -0,0 +1,58 @@
+import json
+
+import hetio.readwrite
+
+import hetmech.hetmat
+
+format_github_url = 'https://github.com/{repo_slug}/raw/{commit}/{path}'.format
+
+hetnet_urls = {
+    # Figure 2D of Himmelstein & Baranzini
+    # (2015) PLOS Comp Bio. https://doi.org/10.1371/journal.pcbi.1004259.g002
+    'disease-gene-example':
+        format_github_url(
+            repo_slug='dhimmel/hetio',
+            commit='9dc747b8fc4e23ef3437829ffde4d047f2e1bdde',
+            path='test/data/disease-gene-example-graph.json',
+        ),
+    # The bupropion and nicotine dependence Hetionet v1.0 subgraph.
+    'bupropion-subgraph':
+        format_github_url(
+            repo_slug='dhimmel/hetio',
+            commit='30c6dbb18a17c05d71cb909cf57af7372e4d4908',
+            path='test/data/bupropion-CbGpPWpGaD-subgraph.json.xz',
+        ),
+    # A random Hetionet v1.0 subgraph.
+    'random-subgraph':
+        format_github_url(
+            repo_slug='dhimmel/hetio',
+            commit='30c6dbb18a17c05d71cb909cf57af7372e4d4908',
+            path='test/data/random-subgraph.json.xz',
+        ),
+}
+
+
+hetnet_io_cache = {}
+
+
+def get_graph(name, hetmat=False, directory=None):
+    """
+    If hetmat=True, import graph into a hetmat located on-disk at directory.
+    """
+    if name not in hetnet_urls:
+        raise ValueError(
+            f'{name} is not a supported test hetnet.\n'
+            'Choose from the following currently defined hetnets: ' +
+            ', '.join(hetnet_urls)
+        )
+    if name not in hetnet_io_cache:
```

Comment:
In other files you import just this function. Does the hetnet_io_cache, defined in this file, get transferred to others?

Reply:
When you import a function, it retains the scope of its source file.